src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,97 @@
+package duke;
+
+import duke.command.Command;
+import duke.exceptions.DukeException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.Todo;
+import duke.ui.Ui;
+
+import java.util.Scanner;
```

Comment:
Perhaps a better way to write import statements is to order them on:
1. Java Packages, then Custom Packages
2. Lexical Order